packages/client-sdk-nodejs/src/config/configurations.ts

```diff
@@ -154,6 +157,47 @@ class InRegionLowLatency extends CacheConfiguration {
   }
 }
 
+class InRegionLambda extends CacheConfiguration {
```

Comment:
I don't think we need to call this `InRegion` (even though technically it almost certainly always will be).

Here's the prior art we have on naming choices for this:

https://github.com/momentohq/client-sdk-dotnet/blob/0022707888f8e288b352c90c07221d462131fa12/src/Momento.Sdk/Config/Configurations.cs#L172

Reply:
I'm following the naming pattern of the other configurations in this class. We use InRegionDefault and InRegionLowLatency. The export class InRegion renames them so we can specify them as Configurations.InRegion.Default.

Reply:
what i'm saying is that in the .NET sdk, the `Lambda` configuration is entirely distinct from the `InRegion` ones.